iosHackthonApp/iosHackthonApp.xcodeproj/xcshareddata/xcschemes/iosHackthonApp.xcscheme

```diff
@@ -28,6 +28,16 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
+         <TestableReference
```

Comment:
This adds the tests to the main scheme so you can run tests locally easier, without having to switch schemes. The tests will only build if you hit cmd+u or build for testing with cmd+shift+u, so it won't affect build times for regular builds